scripts/podman-volume-backup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Get a list of all volumes
+volumes=$(podman volume ls -q)
+#get timestamp for unique filename
+timestamp=$(date +"%Y%m%d%H%M%S")
+
+# Pause all running containers
+echo "Pausing containers for backup"
+podman pause $(podman ps -q)
+
+# Iterate over the volumes
+for volume in $volumes
+do
+#backup the listed vols
+    echo "Backing up ${volume}..."
+    podman volume export ${volume} --output ${volume}.tar
```

Comment:
I think this would be safer to do in a tmpdirectory, and then remove the temp dir when creating podmanbackup_*

Reply:
Normally for a PR unless the changes are completely unrelated to eachother, we just squash them into a single commit.  That makes it easier for other PRs to reference it by a single commit-ID instead of multiple.